presets/ui-kit/src/index.js

```diff
@@ -28,6 +28,11 @@ export { default as Grid } from '@commercetools-uikit/grid';
 export { default as Label } from '@commercetools-uikit/label';
 export { default as LoadingSpinner } from '@commercetools-uikit/loading-spinner';
 export { ErrorMessage, WarningMessage } from '@commercetools-uikit/messages';
+export {
+  PageNavigator,
+  PageSizeSelector,
+  Pagination,
+} from '@commercetools-uikit/pagination';
```

Comment:
You have to specify this as a dependency in `package.json`

Reply:
Hmm, can I do it already on this PR though, since the package is only released after this gets merged?

I think we've been always adding new components to the preset on a follow-up PR.

Reply:
It's a monorepo 🙂
So yes, you should add it now.